USER_MANUAL.md

```diff
@@ -628,7 +673,526 @@ The measured total site level emission rate on January 30th is assumed to have l
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** No data acquisition required.
+
+### &lt;follow_up&gt;
+
+**Description:** The following parameters are used to specify the work practices that enforce the scheduling of the relevant follow-up method .
+
+#### &lt;preferred_method&gt;
+
+**Data type:** String
+
+**Default input:** N/A
+
+**Description:** This parameter allows surveying methods to determine which follow-up method to trigger if multiples are present. If not set, LDAR-Sim will expect only one follow-up method to be present for a single program. Leveraging this parameter enables the use of alternative site or equipment level follow-up methods, facilitating subsequent rounds of screenings based on initial screening results.
+
+Moreover, this feature can model work practices involving multiple screenings to increase confidence in fugitive emissions, measurement accuracy, etc. Each method in the follow-up chain must have the preferred_method set to the corresponding method it triggers.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** If a method within a program is set to have a preferred follow-up method, all methods requiring a follow-up in the program should also be set with a preferred method to avoid ambiguity regarding which follow-up method is being used. Moreover, the last follow-up method in a work practice must be at a component level for leaks to be tagged and repaired. The value of this parameter must be identical to an existing[method name](#method_name).
+
+#### &lt;delay&gt; (follow_up)
+
+**Data type:** Numeric (Integer)
+
+**Default input:** 0
+
+**Description:** The number of days required to have passed since the first site added to the site watch list before a site can be flagged. The company will hold all measurements in a site watch list. The emissions rate used to triage flagging based on follow-up threshold and proportion are specified with [redundancy filter](#redundancy_filter).
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** N/A
+
+#### &lt;instant_threshold&gt;
+
+**Data type:** Numeric (Float)
+
+**Default input:** 0.0
+
+**Description:** The follow-up instant threshold in grams per second. Measured site-level emissions must be above the follow-up threshold before a candidate site becomes immediately available for flagging. If _follow_up.instant_threshold_type_ is "absolute", the numeric value indicates the follow-up threshold in grams per second. If "relative", the numeric value is passed to a function that calculates the emission rate that corresponds to a desired proportion of total emissions for a given leak size distribution. The function estimates the MDL needed to find the top X percent of sources for a given leak size distribution. For example, given a proportion of 0.01 and a leak-size distribution, this function will return an estimate of the follow-up threshold that will ensure that all leaks in the top 1% of leak sizes are found.
```

Comment:
Remove the references to relative here - There is no longer an instant threshold type